content/en/core/overview/roadmap.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Roadmap"
+linkTitle: "Roadmap"
+weight: 1
+description: >
+  Improvements that are being worked on, and those that are being considered next
+aliases:
+  -    /core/overview/technical-roadmap
+relatedContent:
+  core/releases
+---
+
+{{% pageinfo %}}
+The CHT is actively maintained by [Medic](https://medic.org), a non-profit organization that serves as the technical steward, alongside many [partners in the CHT Community]( {{% ref "why-the-cht#the-cht-community" %}}).
```

Comment:
I'm on the fence between "actively developed" and "actively maintained and developed", but leaned towards the former by [borrowing language](https://www.repostatus.org/#active). 
```suggestion
The CHT is actively developed by [Medic](https://medic.org), a non-profit organization that serves as the technical steward, alongside many [partners in the CHT Community]( {{% ref "why-the-cht#the-cht-community" %}}).
```